Week3/homework/project.js

```diff
@@ -0,0 +1,50 @@
+
+"use strict";
+
+function cardValidator(creditCardNumber){
+    let msg = "";
+    const cardNumStr = creditCardNumber.toString();
+    let sum = 0;
+    for(let i = 0; i < cardNumStr.length; i++){
+       sum += parseInt(cardNumStr[i]);
+    }
```

Comment:
Explore arrays `reduce` method here to sum all elements of an array. It is quite nice.